### ISSUE TITLE
[BUG] Add Alt Text to Existing Profile images for Groups

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5548]
**Describe the bug**
This is a bug split from Epic #5547. After consulting with Vanessa, it made sense to split the remediation effort by the affected app areas. 

The scope of this bug covers Profile images located in Groups member lists and Admin Group Settings.

**To Reproduce**
1. Enable Screen reader
2. Go to a 'Group' you are a member of
3. Gesture through items, ensure that images are announcing alt text to user

**Expected behavior**
All non decorative images need alternative text to convey meaning to user. 

**Actual behavior**
The images added before the Alt text field was created all have NULL alt text. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/bfe94ac8-a6f8-4aa3-821c-adfcce6fc2e6)

![Image](https://github.com/user-attachments/assets/052ac39c-a320-442d-9daf-ee5c18b9259c)

This alt text should work regardless of platform or device. Therefore not including info of specific test environments. 

**Additional context**

**Comment by acovher2** [comment]
https://github.com/user-attachments/assets/680aa605-4b7e-4f89-9130-c74095965011
@TodorBachvarov Please see the attached video. I have placed this alt text upon uploading the image and it is not announced in Directory, Not receiving focus on Profile editing page, and not announcing on Groups member list either. I think something went wrong and it is now not reading any alt text. Please advise. 

![Image](https://github.com/user-attachments/assets/b6fc3072-ba72-4857-a8d9-cb37847f9edb)
